### PR TITLE
change flex to grid

### PR DESCRIPTION
**Pull request by DidenkoDenis306**
- [DEMO LINK](https://DidenkoDenis306.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/blocks/_catalog.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.catalog {
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  margin: 0 auto;
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/_catalog.scss]: BEM

File: src/styles/blocks/_catalog.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  @include more488 {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @include more768 {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @include more1024 {
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/_catalog.scss]: Rename
`onMobileLarge`
`onTablet`
`onDesktop`

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
GJ, almost done

**Review by Anna-Tkachenko — APPROVED** [review]
Well Done! ⭐️

**Pull request closed without merging**